### PR TITLE
Event Agenda Topic Tweaks

### PR DESCRIPTION
**Pull request by amaisano**
#### Summary of changes
**Asana Ticket:** [Updating agenda topic struct](https://app.asana.com/0/385363666817452/1200828330950422/f)

Adds the following fields to Agenda Topics
- SubTopic (new paragraph type)
- Links
- Files

Other things:
- New CMS helper function clause for multiple links
- Updated CMS test fixtures (JSON)

---

Before getting review, please check the following:

* [ ] Does frontend functionality render and work correctly in IE?
* [ ] Have we load-tested any new pages or internal API endpoints that will receive significant traffic? [See load testing docs](./../apps/site/load_tests/README.md)
* [ ] Are interactive elements accessible to screen readers?
* [ ] Have you checked for tech debt you can address in the area you're working in?
* [ ] If this change involves routes, does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?
* [ ] Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?

### CODE REVIEW

File: apps/cms/lib/helpers.ex [hunk 1]
```diff
@@ -143,6 +143,15 @@ defmodule CMS.Helpers do
     end
   end
 
+  # This is silly and should be combined/refactored with above clause
```
[the remark this answers is not in the document]
Comment (thecristen) [reply apps/cms/lib/helpers.ex:146]: Ha that's kinda tricky! Maybe something like this (untested), but that's a bit of a refactor 🤷🏼‍♀️ 

```elixir
  @spec parse_link_or_links(map, String.t()) :: [Link.t()] | Link.t() | nil
  def parse_link_or_links(%{} = data, field) when :erlang.is_map_key(field, data)  do
    case data[field] do
      [link] -> Link.from_api(link) # or [Link.from_api(link)] if we always want to return a list, which I kinda like
      [_, _] = links -> Enum.map(links, &Link.from_api(&1))
      _ -> nil
    end
  end
  
  def parse_link_or_links(_, _), do: nil
```

**Review by thecristen — APPROVED** [review]
Looking forward to using this new data, which we get get into in the followup CSS ticket.

**Comment by amaisano** [comment]
@thecristen I agree that it's probably good fodder for a refactor, and to always expect lists. I didn't want two different functions, and i didn't want a function that could return a list or not a list.

The question is, should we ship this now and refactor later? Refactoring will bring this into next week for me I think, and could potentially cause regressions.

**Comment by hannahpurcell** [comment]
> @thecristen I agree that it's probably good fodder for a refactor, and to always expect lists. I didn't want two different functions, and i didn't want a function that could return a list or not a list.
> 
> The question is, should we ship this now and refactor later? Refactoring will bring this into next week for me I think, and could potentially cause regressions.

**Pull request merged**